scss/components/_posts.scss

```diff
@@ -2,6 +2,7 @@
 
 .post-feed {
   border-top: 1px solid $gray;
+  border-bottom: 1px solid $gray;
   padding: u(1rem 0 1rem 2rem);
```

Comment:
Yep, that's the style we're going with as it's used on the press landing page too.